### DISCUSSION TITLE
Does django-rest-framework support nested serializers with multipart data ?

### DISCUSSION THREAD

**Discussion by iamMHZ** [encode/django-rest-framework#8053] [Question & Answer]
There is a closed issue [here](https://github.com/encode/django-rest-framework/issues/7262), and I was wondering if there is a way to deal with nested serializers that have multipart data? Does DRF come with an approach out of the box?

**Comment by lovelydinosaur** [comment]
I think this comment in particular is the closest to what you're looking for https://github.com/encode/django-rest-framework/issues/7262#issuecomment-737364846
